client/src/components/LoginPage/index.js

```diff
@@ -83,10 +102,14 @@ const LoginPage = (props) => {
 
   return (
     <div>
-      <h1 className="Title">Login</h1>
+      <h1 className="Title">
+        <StyledText>Login</StyledText>
+      </h1>
       <div className="box">
         <center>
```

Comment:
If you want the text to be centered, rather than do
```
<center>
<h1>
<StyledText>
</StyledText>
</h1>
</center>
```

You can just add "text-align: center;" to StyledText on line 24 and just do 
```
<StyledText> text </StyledText>
```

Reply:
fixed